realsense_ros2_camera/src/realsense_camera_node.cpp

```diff
@@ -495,6 +499,7 @@ class RealSenseCameraNode : public rclcpp::Node
 
           if (_pointcloud && is_depth_frame_arrived && is_color_frame_arrived) {
```

Comment:
I would suggest use a separated option for "align_depth". Make it default true, the same as ROS Realsense. "enable_pointcloud" should be used for the other feature.